### ISSUE TITLE
Error in GUI!

### ISSUE THREAD

**Issue by derrapf** [ioBroker/ioBroker.admin#1445]
IOBroker came up with this page when I wanted to show the instances:
![image](https://user-images.githubusercontent.com/11745094/165628751-97058bcb-d830-4137-9b1c-0c4eaaefdcb8.png)

**To Reproduce**  
1. Go to instances'
2. See error
3. Refresh the (F5)
4. Everything is Ok.

**Expected behavior**  
Show the instances

**Screenshots & Logfiles**  
See above
F12 shows
```
iobroker.admin@5.3.8
instrument.ts:124 [2022-04-27T20:46:38.513Z] Try to connect
sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7:1          Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7:1          Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7:1          Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7:1          Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
:8081/static/js/1.f7e5d567.chunk.js:1          Failed to load resource: net::ERR_NETWORK_CHANGED
:8081/static/js/10.25e9b20a.chunk.js:1          Failed to load resource: net::ERR_NETWORK_CHANGED
instrument.ts:124 ChunkLoadError: Loading chunk 1 failed.
(error: http://192.168.111.176:8081/static/js/1.f7e5d567.chunk.js)
    at Function.u.e (VM16:1:3974)
    at App.js:67:36
    at S (react.production.min.js:18:52)
    at Zi (react-dom.production.min.js:269:22)
    at jl (react-dom.production.min.js:250:347)
    at Vl (react-dom.production.min.js:250:278)
    at Cl (react-dom.production.min.js:250:138)
    at bl (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Ga (react-dom.production.min.js:122:325)
    at Ka (react-dom.production.min.js:123:61)
    at Za (react-dom.production.min.js:122:428)
    at ke (react-dom.production.min.js:292:101)
    at Yt (react-dom.production.min.js:73:352)
    at HTMLDivElement.a (helpers.ts:99:17)
(anonymous) @ instrument.ts:124
sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7:1          Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
instrument.ts:124 [2022-04-27T20:48:05.221Z] ws connection error: CLOSE_ABNORMAL
(anonymous) @ instrument.ts:124
instrument.ts:124 [2022-04-27T20:48:05.750Z] Start reconnect 0
instrument.ts:124 [2022-04-27T20:48:05.757Z] Try to connect
DevTools failed to load source map: Could not load content for chrome-extension://mmhlniccooihdimnnjhamobppdhaolme/lib/tp/browser-polyfill.min.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
```

**Versions:**  
 - Adapter version: <adapter-version>
 - JS-Controller version:   3.3.22 
 - Node version:v12.22.6
 - System Information:
Rock64
Plattform: linux
Betriebssystem: linux
Architektur: arm64
CPUs: 4
Geschwindigkeit: 1296 MHz
RAM: 3.88 GB
Node.js_ v12.22.6 
NPM: 6.14.15
Datenträgergröße: 57.77 GB
freier Festplattenspeicher: 45.73 GB
Anzahl der Adapter: 407
Aktive Instanzen: 38

**Additional context**

**Comment by GermanBluefox** [comment]
This error should be fixed in 5.4.7

**Comment by mcuiobroker** [comment]
Error in GUI v5.49:
```
TypeError: Cannot read properties of undefined (reading 'common')
    at s.value (ObjectBrowser.js:4627:49)
    at s.value (ObjectBrowser.js:4708:23)
    at Go (react-dom.production.min.js:187:188)
    at Uo (react-dom.production.min.js:186:173)
    at Zi (react-dom.production.min.js:269:427)
    at jl (react-dom.production.min.js:250:347)
    at Vl (react-dom.production.min.js:250:278)
    at Cl (react-dom.production.min.js:250:138)
    at bl (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Ga (react-dom.production.min.js:122:325)
    at Ka (react-dom.production.min.js:123:61)
    at Za (react-dom.production.min.js:122:428)
    at pl (react-dom.production.min.js:237:203)
    at Object.enqueueForceUpdate (react-dom.production.min.js:134:12)
    at s.m.forceUpdate (react.production.min.js:12:456)
    at ObjectBrowser.js:2230:26
    at a (helpers.ts:99:17)
(anonym) @ instrument.ts:124
```
